code/datums/storage/subtypes/medkits.dm

```diff
@@ -0,0 +1,142 @@
+///Regular medkit
+/datum/storage/medkit
+	open_sound = 'sound/items/handling/medkit/medkit_open.ogg'
+	open_sound_vary = TRUE
+	rustle_sound = 'sound/items/handling/medkit/medkit_rustle.ogg'
+	max_specific_storage = WEIGHT_CLASS_SMALL
+
+	///List of everything a medkit can hold
+	VAR_FINAL/static/list/obj/item/list_of_everything_medkits_can_hold = list(
+		//surgery tools
+		/obj/item/surgical_drapes,
+		/obj/item/scalpel,
+		/obj/item/circular_saw,
+		/obj/item/bonesetter,
+		/obj/item/surgicaldrill,
+		/obj/item/retractor,
+		/obj/item/cautery,
+		/obj/item/hemostat,
+		/obj/item/blood_filter,
+
+		//special tools
+		/obj/item/healthanalyzer,
+		/obj/item/dnainjector,
+		/obj/item/lazarus_injector,
+		/obj/item/implantcase,
+		/obj/item/implant,
+		/obj/item/implanter,
+
+		//stacks
+		/obj/item/stack/medical,
+		/obj/item/stack/medical/gauze,
+		/obj/item/stack/sticky_tape,
+		/obj/item/stack/medical/suture/medicated,
+		/obj/item/stack/medical/mesh/advanced,
+
+		//containers
+		/obj/item/reagent_containers/dropper,
+		/obj/item/reagent_containers/cup/beaker,
+		/obj/item/reagent_containers/cup/bottle,
+		/obj/item/reagent_containers/cup/tube,
+		/obj/item/reagent_containers/applicator/pill,
+		/obj/item/reagent_containers/syringe,
+		/obj/item/reagent_containers/medigel,
+		/obj/item/reagent_containers/spray,
+		/obj/item/reagent_containers/applicator/patch/libital,
+		/obj/item/reagent_containers/applicator/patch/aiuri,
+		/obj/item/reagent_containers/applicator/patch/synthflesh,
```

Comment:
Is there a particular reason we have `/obj/item/reagent_containers/applicator/pill`, `/obj/item/reagent_containers/applicator/patch/libital`, `/obj/item/reagent_containers/applicator/patch/aiuri`, and `/obj/item/reagent_containers/applicator/patch/synthflesh` here when the previous medkit list just had the parent type `/obj/item/reagent_containers/applicator`?

Reply:
I don't know why I did this. Good point I'll remove the subtypes